msgid ""
msgstr ""
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"X-Crowdin-Project: orchard-core\n"
"X-Crowdin-Project-ID: 307044\n"
"X-Crowdin-Language: fr\n"
"X-Crowdin-File: /main/Localization/OrchardCore.Setup.Core.pot\n"
"X-Crowdin-File-ID: 812\n"
"Project-Id-Version: orchard-core\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Language-Team: French\n"
"Language: fr_FR\n"
"PO-Revision-Date: 2022-10-11 20:51\n"

#. context.Errors.Add(String.Empty, S["DatabaseProvider setting is required."]);
#: OrchardCore.Setup.Core\SetupService.cs:177
msgctxt "OrchardCore.Setup.Services.SetupService"
msgid "DatabaseProvider setting is required."
msgstr "Le paramètre Fournisseur de base de données est requis."

#. context.Errors.Add(String.Empty, S["The provided database provider is not supported."]);
#: OrchardCore.Setup.Core\SetupService.cs:180
msgctxt "OrchardCore.Setup.Services.SetupService"
msgid "The provided database provider is not supported."
msgstr "Le fournisseur de base de données renseigné n'est pas pris en charge."

#. context.Errors.Add(String.Empty, S["The provided connection string is invalid or server is unreachable."]);
#: OrchardCore.Setup.Core\SetupService.cs:183
msgctxt "OrchardCore.Setup.Services.SetupService"
msgid "The provided connection string is invalid or server is unreachable."
msgstr "La chaîne de connexion fournie est invalide ou le serveur est inaccessible."

#. context.Errors.Add(String.Empty, S["The provided database and table prefix are already in use."]);
#: OrchardCore.Setup.Core\SetupService.cs:186
msgctxt "OrchardCore.Setup.Services.SetupService"
msgid "The provided database and table prefix are already in use."
msgstr "La base de données fournie et le préfixe de table sont déjà utilisés."

#. context.Errors.Add(String.Empty, S["An error occurred while initializing the datastore: {0}", e.Message]);
#: OrchardCore.Setup.Core\SetupService.cs:225
msgctxt "OrchardCore.Setup.Services.SetupService"
msgid "An error occurred while initializing the datastore: {0}"
msgstr "Une erreur est survenue lors de l'initialisation du datastore: {0}"

